src/Particle/DistanceTableAA.cpp

```diff
@@ -38,140 +38,136 @@ DistanceTableData* createDistanceTable(ParticleSet& s, int dt_type)
   DistanceTableData* dt=0;
   std::ostringstream o;
   bool useSoA=(dt_type == DT_SOA || dt_type == DT_SOA_PREFERRED);
-  o << "  Distance table for AA: source/target = " << s.getName() << " useSoA =" << useSoA << "\n";
+  o << "  Distance table for similar particles (A-A):" << std::endl;
+  o << "    source/target: " << s.getName() << std::endl;
+  if (useSoA) {
+    o << "    Using structure-of-arrays (SoA) data layout" << std::endl;
+  } else {
+    o << "    Using array-of-structure (AoS) data layout (less efficient than SoA)" << std::endl;
+  }
+
   if(sc == SUPERCELL_BULK)
   {
     if(s.Lattice.DiagonalOnly)
     {
+      o << "    PBC: bulk   Orthorhombic: yes   Type: PPPO" << std::endl;
```

Comment:
Please split all of these codes "PPPO" etc. over at least two lines: (1) "PPP" is related to the system simulated and input file, while (2) the additional letters relate to the implementation. I prefer a less cryptic long form description "Using optimized orthorhombic implementation". This would also be consistent with the SoA text.

Reply:
I still prefer it to be the same line and short. PPP info is already reflected as "PBC: bluk". O, the actual implementation, doesn't not need to be explained in the output file. It is better to mention it in the manual. Keep them in the same line make it easy for grep.

Reply:
I don't want to ever have to explain these abbreviations e.g. at a school or in the manual. I want them to be separate and that is final.

Reply:
The boundary conditions are already printed out in the 'Simulation Cell' section.  What about a purely text line, like
```
Distance computations use orthorhombic periodic cell in 3D.
```

Reply:
That is good. Just be sure to print DIM-D not just 3D.